edx/edx-ora2
# feat: allow some staffworkflow fields to be blank

**TL;DR -** `scorer_id`, `grading_completed_at`, `grading_started_at`, `cancelled_at`, and `assessment` should be allowed to be blank

JIRA: [AU-428](https://openedx.atlassian.net/browse/AU-428)

**Developer Checklist**


**Testing Instructions**

Install ORA locally and navigate to https://internal.courses.edx.org/admin/assessment/staffworkflow/ and choose an unscored workflow. click save without changing anything. It should save correctly! set one of the datetime values and save, then clear and save. both should work.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
